ci/benchmark_ai.sh

```diff
@@ -0,0 +1,218 @@
+ /mxnet-model-server/docker/mms_app_gpu.conf
+sed -i -e 's/\/usr\/local\/Cellar\/jmeter\/3.3\/libexec\/lib\/ext\/CMDRunner.jar/\/home\/ubuntu\/apache-jmeter-4.0\/lib\/ext\/CMDRunner.jar /g' /mxnet-model-server/load-test/run_load_test.sh
+}
+
+
+## parse logs
+parse_csv_log(){
+    echo "$report"
+    OLDIFS= $IFS
+    IFS=","
+    read -r sampler_label aggregate_report_count average aggregate_report_median aggregate_report_90_line aggregate_report_95_line aggregate_report_99_line aggregate_report_min aggregate_report_max a\
+ggregate_report_error aggregate_report_rate aggregate_report_bandwidth aggregate_report_stddev < <(tail -n1 /mxnet-model-server/load-test/report/$report)
+
+    echo -e "{\
+        Throughput_concurrency_${workers}_req_${requests} :$aggregate_report_rate,\
+        Average_latency_concurrency_${workers}_req_${requests} :$average,\
+        Median_latency_concurrency_${workers}_req_${requests} :$aggregate_report_median,\
+        P90_latency_concurrency_${workers}_req_${requests} :$aggregate_report_90_line,\
+        Error_rate_concurrency_${workers}_req_${requests} :$aggregate_report_error}\n"
+        IFS=$OLDIFS
+}
+
+#start the container test
+start_mms_cpu(){
+    docker run  --name mms -v /mxnet-model-server:/mxnet-model-server  -itd -p 80:8080 awsdeeplearningteam/mms_cpu
+    docker exec mms pip install -U -e /mxnet-model-server/.  >& /dev/null
+    docker exec mms mxnet-model-server start --mms-config /mxnet-model-server/docker/mms_app_cpu.conf >& /dev/null &
+    if [[ `echo $?` != 0 ]] ; then
+        echo "ERROR: Exiting as MMS startup failed"
+        remove_container_images
+        exit 1
+    fi
+    sleep 30
+}
+
+start_mms_gpu(){
+    nvidia-docker run --name mms -v /mxnet-model-server:/mxnet-model-server -itd -p 80:8080 awsdeeplearningteam/mms_gpu
+    nvidia-docker exec mms pip install -U -e /mxnet-model-server/. >& /dev/null
+    nvidia-docker exec mms pip uninstall --yes mxnet-cu90mkl  >& /dev/null
+    nvidia-docker exec mms pip uninstall --yes mxnet  >& /dev/null
+    nvidia-docker exec mms pip install mxnet-cu90mkl  >& /dev/null
+    nvidia-docker exec mms mxnet-model-server start --mms-config /mxnet-model-server/docker/mms_app_gpu.conf >& /dev/null &
+    if [[ `echo $?` != 0 ]] ; then
+        echo "ERROR: Exiting as MMS startup failed"
+        remove_container_images
+        exit 1
+    fi
+    sleep 90
+}
+
+start_jmeter(){
+    cd /mxnet-model-server/load-test
+    echo "$report"
+    ./run_load_test.sh -i 127.0.0.1 -c $workers -n $requests -f $report -p 80
+    if [[ `echo $?` != 0 ]] ; then
+        echo "ERROR: Exiting as jmeter startup failed"
+        remove_container_images
+        exit 1
+    fi
+    echo "done report generation"
+}
+
+remove_container_images(){
+    docker rm -f mms
+}
+
+run_test(){
+    export JMETER_HOME=$HOME/apache-jmeter-4.0
+    export PATH=$JMETER_HOME/bin:$PATH
+    for workers in 50 100
```

Comment:
This also seems like hard-coded value. You could take this as variable array at the top of the file as follows
"num_workers=(50 100)"
and you can use num_workers here
 "${num_workers[@]}"

Not a blocker for commit since this is the only place you use this values.